var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -62,7 +62,6 @@ class Relion(CMakePackage, CudaPackage):
 
     # relion 3 supports cuda 9
     # relion < 3 does not
-    depends_on('cuda', when='+cuda')
```

Comment:
You should keep this line. Without it, it becomes more difficult to specify which CUDA version you want (you would have to explicitly state the `relion` version). This is a workaround for a bug in Spack's concretization mechanism.